net-misc/ntp/ntp-4.2.8p11.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A network protocol suite"
+DESCRIPTION="network time protocol suite/programs"
+HOMEPAGE="http://www.ntp.org"
+COPYRIGHT="1992-2015 University of Delaware
+	2011-2017 Network Time Foundation"
+LICENSE="NTP"
+REVISION="1"
+SOURCE_URI="https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-$portVersion.tar.gz"
+CHECKSUM_SHA256="f14a39f753688252d683ff907035ffff106ba8d3db21309b742e09b5c3cd278e"
+PATCHES="ntp-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 !x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	ntp$secondaryArchSuffix
+	cmd:ntp$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:which
+	"
+
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure ./configure --disable-simulator --without-sntp --without-ntpsnmpd --enable-step-slew
```

Comment:
```
	runConfigure ./configure --disable-simulator --without-sntp --without-ntpsnmpd --enable-step-slew
```
This line shoukd probably be split in 2 or more lines. What do you think of:
```
	runConfigure ./configure --disable-simulator --without-sntp \
		--without-ntpsnmpd --enable-step-slew
```
or
```
	runConfigure ./configure \
		--disable-simulator \
		--enable-step-slew \
		--without-ntpsnmpd \
		--without-sntp \

```